docs/_articles/en/v3.x/custom-rules/author.md

```diff
@@ -16,27 +16,14 @@ Because PMD and ESLint custom rules work differently, Code Analyzer deals with e
 
 ### Write PMD Custom Rules
 
-To be compatible with Code Analyzer, your PMD custom rules must meet these criteria.
+To be compatible with Code Analyzer, your PMD custom rules must meet these guidelines.
```

Comment:
I think we decided we were going to reword this as:
"To be compatible with Code Analyzer, build your PMD custom rules following these guidelines:"